freemocap/fmc_skeleton_data_holder.py

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
Is this related to the `origin alignment` code? 

If not, it should technically be in a different pull request. 

(obviously we haven't been that careful about keeping our commits and PRs separated by 'feature' like that, but we'll want to start thinking in that was as we move forward with this project)

Reply:
yes, it's used in the origin alignment.